client-js/main/client/actor-request-factory.js

```diff
@@ -815,7 +815,7 @@ class QueryFactory {
    */
   static _newId() {
     const result = new QueryId();
-    result.setValue(`q-${uuid.v4()}`);
+    result.setValue(`q-${uuidv4()}`);
```

Comment:
Is this some kind of a convention? Why do we alias this particular import?

Reply:
The default exported item is `v4` and I don't think it worth using it as is. That's why I used the alias.

With just `q-${v4()}` it is more awkward.